manifests/05-operator.yaml

```diff
@@ -41,10 +41,10 @@ spec:
           value: docker.io/openshift/origin-console:latest
         - name: OPERATOR_NAME
           value: "console-operator"
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
```

Comment:
Was removing this intentional?

Reply:
no it wasnt... probably caused by switching between the branches. Should be good now